### PR TITLE
Migration: winforms & wpa group updates

### PR DESCRIPTION
**Pull request by guardrex**
ready for :eyes:

### CODE REVIEW

File: docs/framework/configure-apps/file-schema/winforms/windows-forms-add-configuration-element.md [hunk 1]
```diff
@@ -33,33 +33,33 @@ The following sections describe attributes, child elements, and parent elements.
 
 ### Attributes
```
Comment (mairaw) [inline docs/framework/configure-apps/file-schema/winforms/windows-forms-add-configuration-element.md]: i've noticed this topic and some others are missing the ms.prod value which is a required metadata. we should file an issue to take care of that.

File: docs/framework/winforms/high-dpi-support-in-windows-forms.md [hunk 2]
```diff
@@ -107,7 +107,7 @@ The .NET Framework 4.7 also adds a number of new helper methods and properties t
 
 ## Versioning considerations 
 
-In addition to running on .NET Framework 4.7 and Windows 10 Creators Update, your application may also run in an environment in which it is not compatible with high DPI improvements. In this case, you'll need to develop a fallback for your application. You can do this to perform [custom drawing](User-Drawn%20Controls.md) to handle scaling. 
+In addition to running on .NET Framework 4.7 and Windows 10 Creators Update, you application may also run in an environment in which it's not compatible with high DPI improvements. In this case, you'll need to develop a fallback for your application. You can do this to perform [custom drawing](../../../docs/framework/winforms/controls/user-drawn-controls.md) to handle scaling.
```
Comment (mairaw) [inline docs/framework/winforms/high-dpi-support-in-windows-forms.md]: your application seemed correct here

File: docs/framework/wpf/advanced/the-ink-object-model-windows-forms-and-com-versus-wpf.md [hunk 3]
```diff
@@ -55,7 +55,7 @@ There are essentially three platforms that support digital ink: the Tablet PC Wi
 |Task|Windows Presentation Foundation|Windows Forms and COM|  
 |----------|-------------------------------------|---------------------------|  
 |Save Ink|<xref:System.Windows.Ink.StrokeCollection.Save%2A>|[Microsoft.Ink.Ink.Save](https://technet.microsoft.com/library/security/microsoft.ink.ink.save(v=vs.90))|  
-|Load Ink|Create a <xref:System.Windows.Ink.StrokeCollection> with the <xref:System.Windows.Ink.StrokeCollection.ctor> constructor.|<xref:Microsoft.Ink.Ink.Load%2A>|  
+|Load Ink|Create a <xref:System.Windows.Ink.StrokeCollection> with the <xref:System.Windows.Ink.StrokeCollection.%23ctor%2A> constructor.|[Microsoft.Ink.Ink.Load](https://msdn.microsoft.com/library/microsoft.ink.ink.load.aspx)|
```
Comment (mairaw) [inline docs/framework/wpf/advanced/the-ink-object-model-windows-forms-and-com-versus-wpf.md]: wondering if we should perhaps add this api to the https://github.com/dotnet/docs/blob/master/_zip/missingapi.yml file. Or at least I'd link to version that is linked to the TOC: https://msdn.microsoft.com/en-us/library/microsoft.ink.ink.load(v=vs.90).aspx

File: docs/framework/configure-apps/file-schema/winforms/windows-forms-add-configuration-element.md (hunk 1 above)
Comment (GuardRex) [inline docs/framework/configure-apps/file-schema/winforms/windows-forms-add-configuration-element.md]: https://github.com/dotnet/docs/issues/2061

File: docs/framework/winforms/high-dpi-support-in-windows-forms.md (hunk 2 above)
Comment (GuardRex) [inline docs/framework/winforms/high-dpi-support-in-windows-forms.md]: yep ... typo ... thx

File: docs/framework/wpf/advanced/the-ink-object-model-windows-forms-and-com-versus-wpf.md (hunk 3 above)
Comment (GuardRex) [inline docs/framework/wpf/advanced/the-ink-object-model-windows-forms-and-com-versus-wpf.md]: Link updated. Noted API in Migration Project for additional work.

**Review by rpetrusha — CHANGES_REQUESTED** [review]
@Guardrex, can you add the top-level file-schema/winforms/index.md to your PR? The syntax section for the `ConfigurationSection` element has `\<` for both the begin and end tags. Also, the first entry int he "See also" section needs to be terminated by 3 spaces. Also, the element name is System.Windows.Forms.ApplicationConfigurationSection.

**Comment by GuardRex** [comment]
@rpetrusha 

> add the top-level file-schema/winforms/index.md

... in *docs\framework\configure-apps\toc.md*? If so, then updates are complete.

File: docs/framework/configure-apps/file-schema/winforms/windows-forms-add-configuration-element.md [hunk 4]
```diff
@@ -91,11 +92,11 @@ High DPI awareness is an opt-in feature; by default, the value of `DpiAwareness`
 The single default opt-in setting provided by setting `DpiAwareness` to `PerMonitorV2` is generally adequate for new Windows Forms applications. However, You can then opt out of individual high DPI improvements by adding the corresponding key to the application configuration file. For example, to take advantage of all the new DPI featuers except for dynamic DPI support, you would add the following to your application configuration file:
 
 ```xml
-<System.Windows.Forms.ApplicationConfigurationSection>
-  <add key="DpiAwareness" value="PerMonitorV2" />
-  <!-- Disable dynamic DPI support -->
-  <add key="EnableWindowsFormsHighDpiAutoResizing" value"false" />
-</System.Windows.Forms.ApplicationConfigurationSection>
+<System.Windows.Forms.ConfigurationSection>
```
Comment (rpetrusha) [inline docs/framework/configure-apps/file-schema/winforms/windows-forms-add-configuration-element.md]: The original is correct in both cases (ApplicationConfigurationSection, not ConfigurationSection)

File: docs/framework/winforms/high-dpi-support-in-windows-forms.md [hunk 5]
```diff
@@ -37,52 +37,52 @@ In addition, to configure high DPI support in your Windows Forms application, yo
...
-		   \<!-- Windows 10 compatibility -->
-		   <supportedOS Id="{8e0f7a12-bfb3-4fe8-b9a5-48fd50a15a9a}" />
-	   </application>
-   </compatibility>
-   ```
 - Enable per-monitor DPI awareness in the app.config file.
 
-   Windows Forms introduces a new [\<System.Windows.Forms.ApplicationConfigurationSection>]
-(Windows%20Forms%20Configuration%20Section.md) to support new features and customizations added starting with the .NET Framework 4.7. To take advantage of the new features that support high DPI, add the following to your application configuration file.   
+  Windows Forms introduces a new [`<System.Windows.Forms.ConfigurationSection>`](../../../docs/framework/configure-apps/file-schema/winforms/index.md) element to support new features and customizations added starting with the .NET Framework 4.7. To take advantage of the new features that support high DPI, add the following to your application configuration file.
```
Comment (rpetrusha) [inline docs/framework/winforms/high-dpi-support-in-windows-forms.md]: ApplicationConfigurationSection is correct.

File: docs/framework/winforms/high-dpi-support-in-windows-forms.md [hunk 6]
```diff
@@ -37,52 +37,52 @@ In addition, to configure high DPI support in your Windows Forms application, yo
...
-(Windows%20Forms%20Configuration%20Section.md) to support new features and customizations added starting with the .NET Framework 4.7. To take advantage of the new features that support high DPI, add the following to your application configuration file.   
+  Windows Forms introduces a new [`<System.Windows.Forms.ConfigurationSection>`](../../../docs/framework/configure-apps/file-schema/winforms/index.md) element to support new features and customizations added starting with the .NET Framework 4.7. To take advantage of the new features that support high DPI, add the following to your application configuration file.   
 
-   ```xml
-   \<System.Windows.Forms.ApplicationConfigurationSection>
-      <add key="DpiAwareness" value="PerMonitorV2" />
-   \</System.Windows.Forms.ApplicationConfigurationSection>	   
-   ```
+  ```xml
+  <System.Windows.Forms.ConfigurationSection>
```
Comment (rpetrusha) [inline docs/framework/winforms/high-dpi-support-in-windows-forms.md]: `ApplicationConfigurationSection ` is correct in both cases.

File: docs/framework/winforms/high-dpi-support-in-windows-forms.md [hunk 7]
```diff
@@ -37,52 +37,52 @@ In addition, to configure high DPI support in your Windows Forms application, yo
...
 ## Opting out of individual high DPI features
 
 Setting the `DpiAwareness` value to `PerMonitorV2` enables all high DPI awareness features supported by .NET Framework versions starting with the .NET Framework 4.7. Typically, this is adequate for most Windows Forms applications. However, you may want to opt out of one or more individual features. The most important reason for doing this is that your existing application code already handles that feature.  For example, if your application handles auto scaling, you might want to disable the auto-resizing feature as follows:
 
 ```xml
-\<System.Windows.Forms.ApplicationConfigurationSection>
-   <add key="DpiAwareness" value="PerMonitorV2" />
-   <add key="EnableWindowsFormsHighDpiAutoResizing" value="false" /> 
-\</System.Windows.Forms.ApplicationConfigurationSection>	   
+<System.Windows.Forms.ConfigurationSection>
```
Comment (rpetrusha) [inline docs/framework/winforms/high-dpi-support-in-windows-forms.md]: The original `ApplicationConfigurationSection` is correct in both cases.

**Review by rpetrusha — CHANGES_REQUESTED** [review]
The major issue here is that the documentation was inconsistent in the name of the config element. The correct name is `System.Windows.Forms.ApplicationConfigurationSection`.

**Comment by GuardRex** [comment]
@rpetrusha Updates complete.

File: docs/framework/configure-apps/file-schema/winforms/index.md [hunk 8]
```diff
@@ -20,9 +20,9 @@ Windows Forms application configuration settings are stored in an application co
 
 ```xml
 <configuration>
-  \<System.Windows.Forms.ConfigurationSection>
+  <System.Windows.Forms.ConfigurationSection>
```
Comment (rpetrusha) [inline docs/framework/configure-apps/file-schema/winforms/index.md]: ApplicationConfigurationSection for both line 23 and line 25

**Review by rpetrusha — COMMENTED** [review]
Sorry, @GuardRex, I missed one ConfigurationSection -> ApplicationConfigurationSection.

File: docs/framework/configure-apps/file-schema/winforms/index.md (hunk 8 above)
Comment (GuardRex) [inline docs/framework/configure-apps/file-schema/winforms/index.md]: Lines 17, 51, and 53 look like they need this change, too.

Comment (GuardRex) [inline docs/framework/configure-apps/file-schema/winforms/index.md]: ... and same thing with lines 68 and 70 in *windows-forms-add-configuration-element.md*.

File: docs/framework/configure-apps/toc.md [hunk 9]
```diff
@@ -9,3 +9,4 @@
 ### [Mapping Object Identifiers to Cryptography Algorithms](map-object-identifiers-to-cryptography-algorithms.md)
 ## [How to: Create a Publisher Policy](how-to-create-a-publisher-policy.md)
 ## [Configuration file schema](file-schema/index.md)
+## [Windows Forms Configuration Section](winforms/index.md)
```
Comment (rpetrusha) [inline docs/framework/configure-apps/toc.md]: This should be removed -- Windows Forms configuration section is a child of Configuration file schema, and it's already listed in the Configuration file schema topic.

Comment (rpetrusha) [inline docs/framework/configure-apps/toc.md]: This should be removed. The topic is a parent of the Configuration file schema topic, and it already appears both in the TOC and as a link from that topic.

**Review by rpetrusha — APPROVED** [review]
@GuardRex, aside from my comment about the TOC entry, this looks good. After you address the TOC issue, you can merge this when the build completes.

**Pull request merged**